README.md

```diff
@@ -312,6 +312,38 @@ The following sample plugins exist for NRI:
 Please see the documentation of these plugins for further details
 about what and how each of these plugins can be used for.
 
+## Security Considerations
+
+From a security perspective NRI plugins should be considered part of the
+container runtime. NRI does not implement granular access control to the
+functionality it offers. Access to NRI is controlled by restricting access
+to the systemwide NRI socket. If a process can connect to the NRI socket
+and send data, it has access to the full scope of functionality available
+via NRI.
```

Comment:
I think since this is discussing security, it's better to err on the side of being more verbose and explicit.  Assuming that someone hasn't read enough of the readme to comprehend the "full scope of functionality", we can spell it out a bit.  Here's a possible set of things to describe:

* injection of OCI hooks, which allow for arbitrary execution of processes with the same privilege level as the container runtime
* arbitrary changes to mounts, including new bind-mounts, changes to the proc, sys, mqueue, shm, and tmpfs mounts
* the addition or removal of arbitrary devices
* arbitrary changes to the limits for memory, CPU, block I/O, and (on Intel systems) RDT resources available, including the ability to deny service by setting limits very low

Reply:
Updated, spelling out explicitly all of the above mentioned possibilities.